Tests/SentryTests/SentryCrash/SentryCrashReportSinkTests.swift

```diff
@@ -40,10 +40,14 @@ class SentryCrashReportSinkTests: SentrySDKIntegrationTestsBase {
         
         let reportSink = fixture.sut
         reportSink.filterReports([report]) { _, _, _ in
-            self.assertCrashEventWithScope { _, scope in
-                let data = scope?.serialize()
-                XCTAssertEqual(data?["environment"] as? String, "testFilterReports_CopyHubScope")
-                expect.fulfill()
+            do {
+                try self.assertCrashEventWithScope { _, scope in
+                    let data = scope?.serialize()
+                    XCTAssertEqual(data?["environment"] as? String, "testFilterReports_CopyHubScope")
+                    expect.fulfill()
+                }
+            } catch {
+                XCTFail("Failed to assert crash event with scope: \(error)")
             }
```

Comment:
`m`: Please look into using the `rethrows` keyword for the callback, i.e. [here](https://www.avanderlee.com/swift/rethrows/). As the closure is not using `try` inside, it might be possible to drop the `try` outside too, therefore no need to add the do-catch (should also remove the Codecov branch coverage warning)